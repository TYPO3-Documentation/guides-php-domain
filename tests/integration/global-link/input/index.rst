===================
PHP Global Variable
===================

..  php:global:: $GLOBALS['TCA']

    Contains the TYPO3 Configuration Array

See also :php:global:`$GLOBALS['TCA']`.
