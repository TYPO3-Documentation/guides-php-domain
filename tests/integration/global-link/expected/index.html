<!-- content start -->
    <div class="section" id="php-global-variable">
            <h1>PHP Global Variable</h1>
            <dl class="php global">
    <dt class="sig sig-object php"
        id="globals-tca">
        <em class="property"><span class="pre">global</span> </em>
        <span class="pre">$GLOBALS[&#039;TCA&#039;]</span>
        <a class="headerlink" href="#globals-tca" title="Permalink to this definition">Â¶</a>    </dt>
    <dd>

    <p>Contains the TYPO3 Configuration Array</p>

    </dd>
</dl>

    <p>See also <a href="/index.html#globals-tca">$GLOBALS[&#039;TCA&#039;]</a>.</p>

    </div>
<!-- content end -->
